src/octoprint/plugins/softwareupdate/__init__.py

```diff
@@ -324,8 +325,8 @@ def on_settings_save(self, data):
 
 		if "octoprint_type" in data:
 			octoprint_type = data["octoprint_type"]
-
-			if octoprint_type == "github_release":
+			self._logger.debug("octoprint data - {}".format(data))
+			if octoprint_type == "github_release" or octoprint_type == "github_commit":
```

Comment:
Interesting that this was required. However, to shorten this:
` a in [B, C]` instead of `a == B or a == C`

Reply:
Ok will change this. Yes I was wondering as well why this was needed. But it seems like octoprint was not intended to work with github commits